text/src/autogluon/text/text_prediction/mx/models.py

```diff
@@ -0,0 +1,1505 @@
+import numpy as np
```

Comment:
As a general comment on this PR and future PRs: Always try to separate file moving and logic changes into separate PRs. As it stands, this file (along with modules.py) is extremely difficult to review. Due to the specific circumstances of this PR, I won't make this a blocking issue, but this needs to be considered in future PRs, as I will be reluctant to approve PRs that do this going forward.

Additionally, we should try to keep PRs to below 1000 LOC change if possible (and ideally under 500 LOC). We should try to modularize our code changes as much as possible as it becomes harder to test and properly review as the PR gets larger.